### PR TITLE
updated with links and amended

### PR DESCRIPTION
**Pull request by AbuNavsa**
_Add ticket reference to Pull Request title: e.g. 'FS-123: Add content', if there is no ticket prefix with BAU_

### Change description
_A brief description of the pull request_

- [ ] Unit tests and other appropriate tests added or updated
- [ ] README and other documentation has been updated / added (if needed)
- [ ] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")

### How to test
_If manual testing is needed, give suggested testing steps_

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -168,21 +93,11 @@ the following while in your virtual enviroment:
...
-    --port 80
-```
-
-This will initalise this service, using the current created image
+# Builds and Deploys
+Details on how our pipelines work and the release process is available [here](https://dluhcdigital.atlassian.net/wiki/spaces/FS/pages/73695505/How+do+we+deploy+our+code+to+prod)
+## Paketo
+Paketo is used to build the docker image which gets deployed to our test and production environments. Details available [here](https://github.com/communitiesuk/funding-service-design-workflows/blob/main/readmes/python-repos-paketo.md)
+## Copilot
+Copilot is used for infrastructure deployment. Instructions are available [here](https://github.com/communitiesuk/funding-service-design-workflows/blob/main/readmes/python-repos-copilot.md), with the following values for the fund store:
```
Comment (srh-sloan) [inline README.md:101]: please update references to fund store as this is notification

File: README.md [hunk 2]
```diff
@@ -168,21 +93,11 @@ the following while in your virtual enviroment:
...
+- service-name: fsd-fund-store
```
Comment (srh-sloan) [inline README.md:102]: not fund store

File: README.md [hunk 3]
```diff
@@ -168,21 +93,11 @@ the following while in your virtual enviroment:
 Once the above is done you will have autoformatting and pep8 compliance built
```
Comment (srh-sloan) [inline README.md:93]: all this pre commit stuff is covered in the shared readmes - please replace with links

File: README.md [hunk 4]
```diff
@@ -104,51 +70,10 @@ Based on the attributes set on the template during its creation, ensure that all
...
-GOV_NOTIFY_API_KEY
-SENTRY_DSN
-GITHUB_SHA
-
-```
-docker ps -a
-CONTAINER ID   IMAGE                       COMMAND                  CREATED          STATUS                    PORTS                    NAMES
-42633142c619   paketo-demofsd-app          "/cnb/process/web"       8 seconds ago    Up 7 seconds              0.0.0.0:8080->8080/tcp   peaceful_knuth
-```
+Go to relevant service. See example
```
Comment (srh-sloan) [inline README.md:75]: Can you replace this with actual links to each of the other readmes please

**Review by srh-sloan — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -3,46 +3,12 @@
 [![made-with-python](https://img.shields.io/badge/Made%20with-Python-1f425f.svg)](https://www.python.org/)
 [![Code style : black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
 
-Repo for the funding service design notification.
+This repository hosts a Flask web service for a Funding Notification System, specializing in magic link notifications.
```
[the remark this answers is not in the document]
Comment (AbuNavsa) [reply README.md:6]: Have changed to more general "This repository hosts a Flask web service for Funding Service Design Notification System, handling notifications including magic links and emails."

File: README.md (hunk 2 above)
Comment (AbuNavsa) [reply README.md:102]: corrected to fsd-notification etc.

File: README.md (hunk 4 above)
Comment (AbuNavsa) [reply README.md:75]: paketo section with link is in this section now

Comment (srh-sloan) [reply README.md:84]: So the paketo bit is fine, but each type of notification has it's own readme - please link to those as they contain examples of how to use those notifications

**Review by srh-sloan — CHANGES_REQUESTED** [review]

**Review by srh-sloan — DISMISSED** [review]

**Review by srh-sloan — APPROVED** [review]

**Pull request merged**